src/main/java/parking/entities/Plate.java

```diff
@@ -0,0 +1,85 @@
+package parking.entities;
+
+import parking.validators.ProperPlate;
+import parking.entities.enums.Region;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.math.BigDecimal;
+import java.time.Instant;
+
+@Entity
+public class Plate {
```

Comment:
Polecam Lomboka

Reply:
Dlaczego ta klasa ma taką nazwę? Czy to tablica ma kwotę do zapłacenia i to czy jest/nie jest VIP? Jak można to zmienić?

Reply:
j/w będę starał się teraz z niego korzystać. Wkleiłem też swoją drogą do nowego rozwiązania Lomboka.
Nazwa od Registration Plate. W projekcie nie ma wspomnianego systemu logowania etc. więc i nie trzeba rejestrować kierowców.

Jedyne co się liczy to:
- numer rejestracyjny,
- moment włączenia licznika,
- moment zatrzymania licznika,
- status vip,
- waluta,
- opłata.
Wszystko można zamknąć w jednej tabeli. Można by było jednak faktycznie zmienić nazwę na np. Parking Session. Zrobiłem to żeby mi było prościej, ale poprawnie powinno się to podzielić na mniejsze encje tj:
- utworzyć tabelę kierowcy, której kolumny to status VIP, walutę, ew. imię nazwisko;
- utworzyć tabelę rejestracje, której kolumny to nr rejestracyjny;
- utworzyć tabelę sesje, której kolumny to id rejestracji, id kierowcy, start licznika, stop licznika, opłata.